examples/private_set_intersection/README.md

```diff
@@ -0,0 +1,16 @@
+# Private Set Intersection
+
+## To update this example
+
+1. Build the example, including the Wasm module
+1. Push the module to GS via
+   `./scripts/push-example -e private_set_intersection`
+1. Fix the URL and hash in [`./oak_app_manifest.toml`](./oak_app_manifest.toml)
+1. Sign the module with
+
+   ```bash
+   ./scripts/oak_sign sign \
+     --private-key=examples/keys/ed25519/test.key \
```

Comment:
I think we need an explicit comment stating that here we use a Test key that is only used for debugging

Reply:
Done.